### ISSUE TITLE
pyipp print job examples - Need some sights into the print job/ create job & send document docuemntation..

### ISSUE THREAD

**Issue by learner35** [ctalkington/python-ipp#105]
pyipp print job examples - Need some sights into the print job/ create job & send document documentation..

**Comment by ctalkington** [comment]
printing currently isn't supported. mostly informational at this point

**Comment by learner35** [comment]
Thanks! Any roadmap to support it in future? Seems no other python packages
implementing it with out underlying ipptools currently..

**Comment by ctalkington** [comment]
closing due to age but to answer, I'm trying to focus on information gathering, the serialization class is on my radar to get some love but no time commitment yet. If it can get better understanding of the tag names for creating print jobs we can the implement methods to make such easily possible

**Comment by dellorogiulio** [comment]
Hi @ctalkington, seems I figured out how to print a document using this library. I found a missing part  comparing your source with [sealsystems/node-ipp](https://github.com/sealsystems/node-ipp).
 
In sealsystems lib, they used a 'custom' `data` attribute in which they set the file data (see [this example](https://github.com/sealsystems/node-ipp/blob/4de84434ca02d1ebe600793867551412a7a07985/examples/printPDF.js#L24)) and then they add the 'already-byte' [data directly to the encoded message](https://github.com/sealsystems/node-ipp/blob/4de84434ca02d1ebe600793867551412a7a07985/lib/serializer.js#L389). 

so in your `serializer.py', it is needed a  
```python
encoded +=  data['data']
```
just before returning `encoded`.

In my test, a pdf file is correctly printed.

I open a pull-request with a working example.

Let me know if you intend to update this library or if I need to fork it (i need to use this library without patching the source)
